src/aktualizr_secondary/uptane_verification_test.cc

```diff
@@ -1,413 +0,0 @@
-  EXPECT_TRUE(_secondary->putMetadataResp(addTarget("invalid-revision")));
-  EXPECT_FALSE(_secondary->sendFirmwareResp(getCredsToSend()));
-}
-
-TEST_F(OstreeSecondaryUptaneVerificationTest, fullUptaneVerificationInvalidHwID) {
-  EXPECT_FALSE(_secondary->putMetadataResp(addTarget("", "invalid-hardware-id", "")));
-}
-
-TEST_F(OstreeSecondaryUptaneVerificationTest, fullUptaneVerificationInvalidSerial) {
-  EXPECT_FALSE(_secondary->putMetadataResp(addTarget("", "", "invalid-serial-id")));
-}
-
-TEST_F(SecondaryUptaneVerificationTest, fullUptaneVerificationPositive) {
-  EXPECT_TRUE(_secondary->putMetadataResp(_uptane_repo.getCurrentMetadata()));
-  EXPECT_TRUE(_secondary->sendFirmwareResp(getImageData()));
-}
-
-TEST_F(SecondaryUptaneVerificationTest, IncorrectTargetQuantity) {
-  {
-    // two targets for the same ECU
-    _uptane_repo.addImageFile("second_target", _secondary->getHwIdResp().ToString(),
-                              _secondary->getSerialResp().ToString());
-
-    EXPECT_FALSE(_secondary->putMetadataResp(_uptane_repo.getCurrentMetadata()));
-  }
-
-  {
-    // zero targets for the ECU being tested
-    auto metadata =
-        UptaneRepoWrapper().addImageFile("mytarget", _secondary->getHwIdResp().ToString(), "non-existing-serial");
-
-    EXPECT_FALSE(_secondary->putMetadataResp(metadata));
-  }
-
-  {
-    // zero targets for the ECU being tested
-    auto metadata =
-        UptaneRepoWrapper().addImageFile("mytarget", "non-existig-hwid", _secondary->getSerialResp().ToString());
-
-    EXPECT_FALSE(_secondary->putMetadataResp(metadata));
-  }
-}
-
-TEST_F(SecondaryUptaneVerificationTest, InvalidImageFileSize) {
-  EXPECT_TRUE(_secondary->putMetadataResp(_uptane_repo.getCurrentMetadata()));
-  auto image_data = getImageData();
-  image_data->append("\n");
-  EXPECT_FALSE(_secondary->sendFirmwareResp(image_data));
-}
-
-TEST_F(SecondaryUptaneVerificationTest, InvalidImageData) {
-  EXPECT_TRUE(_secondary->putMetadataResp(_uptane_repo.getCurrentMetadata()));
-  auto image_data = getImageData();
-  image_data->operator[](3) = '0';
-  EXPECT_FALSE(_secondary->sendFirmwareResp(image_data));
-}
-
-#ifndef __NO_MAIN__
-int main(int argc, char** argv) {
-  ::testing::InitGoogleTest(&argc, argv);
-
-  if (argc != 2) {
-    std::cerr << "Error: " << argv[0] << " <ostree rootfs path>\n";
-    return EXIT_FAILURE;
-  }
-
-  OstreeSecondaryUptaneVerificationTest::setOstreeRootfsTemplate(argv[1]);
-
-  logger_init();
-  logger_set_threshold(boost::log::trivial::info);
-
-  return RUN_ALL_TESTS();
-}
-#endif
-
-extern "C" OstreeDeployment* ostree_sysroot_get_booted_deployment(OstreeSysroot* ostree_sysroot) {
-  return OstreeSecondaryUptaneVerificationTest::curOstreeDeployment(ostree_sysroot);
-}
-
-extern "C" const char* ostree_deployment_get_csum(OstreeDeployment* ostree_deployment) {
-  return OstreeSecondaryUptaneVerificationTest::curOstreeRootfsRev(ostree_deployment);
-}
```

Comment:
What happened to this test suite? In the old version there were a bunch of non-OSTree-specific Uptane tests. Those still seem useful to me.

Reply:
yes, the total number of tests has actually increased, I just split the initial test file into two, one with all non-ostree tests and another with ostree specific.